client/src/components/SaveIndicator/SaveIndicator.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import useStyles from './useStyles';
+
+interface SIProps {
+  bStatus: boolean;
+}
+
+const SaveIndicator = ({ bStatus }: SIProps): JSX.Element => {
+  const classes = useStyles();
+  return (
+    <>
+      {
+        <div className={classes.saving_container}>
+          {bStatus ? (
```

Comment:
bStatus? boardStatus, might be a bit more descriptive to just put `boardStatus`

Reply:
I was meaning boolean status to let others know its boolean value, but I can make a change if that's more readable.